deploy/kicbase/Dockerfile

```diff
@@ -227,9 +227,7 @@ RUN systemctl enable docker.service
 # making SSH work for docker container
 # based on https://github.com/rastasheep/ubuntu-sshd/blob/master/18.04/Dockerfile
 RUN mkdir /var/run/sshd
-RUN echo 'root:root' |chpasswd
-RUN sed -ri 's/^#?PermitRootLogin\s+.*/PermitRootLogin yes/' /etc/ssh/sshd_config
-RUN sed -ri 's/UsePAM yes/#UsePAM yes/g' /etc/ssh/sshd_config
+RUN sed -ri 's/^#?PermitRootLogin\s+.*/PermitRootLogin no/' /etc/ssh/sshd_config
```

Comment:
You can remove this sshd_config line too, and leave the sshd default: `#PermitRootLogin prohibit-password`

This allows user to talk with their containerd and other niche use cases, while still prohibiting root password.

https://minikube.sigs.k8s.io/docs/handbook/pushing/#6-pushing-directly-to-in-cluster-containerd-buildkitd

The line with "UsePAM" is unrelated to this change, so probably leave it - or remove it in a separate commit ?

Reply:
sounds good ! thank you for pointing it out, I just updated it